internal/helm/helm.go

```diff
@@ -9,11 +9,26 @@ import (
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
 )
 
-var helmCmd = exec.Command("helm", "home")
+var helmVersionCmd = exec.Command("helm", "version", "--short", "--client")
+var helmHomeCmd = exec.Command("helm", "home")
+
+// SupportedVersion returns if the Helm version is supported by Kyma (currently v2.x.x)
+func SupportedVersion() (bool, error) {
+	helmVersionRaw, err := helmVersionCmd.CombinedOutput()
+	if err != nil {
+		return false, err
```

Comment:
Here if the error is that helm is not installed we should not return an error.
If helm is not in the local system at all, Kyma can install normally.